doc/release-notes/release-8.sgml.in

```diff
@@ -30,7 +30,7 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
 <itemize>
-	<item>
+	<item>Build Requirement Changes
```

Comment:
Please do not dilute  "_most important_ _new_ features" index with this entry.

```suggestion
	<item>
```

Reply:
PR version that triggered this change request added a "Build Requirement Changes" entry to the "most important new features" index. There was a corresponding "Build Requirement Changes" section back then. The current PR version still has a misplaced "Build Requirement Changes" index entry but the corresponding section is gone, making it difficult for the reader to find the indexed changes.

Please restore that section. Just do not list it in the "most important new features" index.